libs/common/src/platform/state/user-state.ts

```diff
@@ -6,24 +6,25 @@ import { StateUpdateOptions } from "./state-update-options";
 
 export type CombinedState<T> = readonly [userId: UserId, state: T];
 
-/**
- * A helper object for interacting with state that is scoped to a specific user.
- */
+/** A helper object for interacting with state that is scoped to a specific user. */
 export interface UserState<T> {
-  /**
-   * Emits a stream of data.
-   */
-  readonly state$: Observable<T>;
+  /** Emits a stream of data. Emits null if the user does not have specified state. */
+  readonly state$: Observable<T | null>;
 
-  /**
-   * Emits a stream of data alongside the user id the data corresponds to.
-   */
+  /** Emits a stream of data alongside the user id the data corresponds to. */
```

Comment:
```suggestion
  /** Emits a stream of data alongside the user id the data for that user. */
```

Reply:
Think I'm still struggling to understand the wording of the suggested change 😁

Reply:
Yeah that is still maybe not so good.

```suggestion
  /** Emits a stream of tuples, with the first element being a user id and the second element being the data for that user. */
```

Do you think that helps?

Reply:
Yep, that looks better. Committed!